pom.xml

```diff
@@ -69,9 +69,9 @@ limitations under the License.
         <flink.reuseForks>true</flink.reuseForks>
 
         <!-- dependencies versions -->
-        <flink.version>1.18.1</flink.version>
-        <flink.major.version>1.18</flink.major.version>
-        <flink.shaded.version>17.0</flink.shaded.version>
+        <flink.version>1.19.1</flink.version>
+        <flink.major.version>1.19</flink.major.version>
+        <flink.shaded.version>19.0</flink.shaded.version>
```

Comment:
Seems Flink 1.19.1 still relies on guava31 in older version of `flink.shaded.version` [somehow](https://github.com/apache/flink/blob/5edb5a93061c36e183899415beaf0dab2495b984/flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java#L29) as discussed in FLINK-33705. This change would break existing integrated cases with the following exception:

```
java.lang.NoClassDefFoundError: org/apache/flink/shaded/guava31/com/google/common/collect/Iterables

	at org.apache.flink.configuration.CoreOptions.mergeListsToArray(CoreOptions.java:221)
	at org.apache.flink.configuration.CoreOptions.getParentFirstLoaderPatterns(CoreOptions.java:159)
	at org.apache.flink.runtime.taskexecutor.TaskManagerServicesConfiguration.fromConfiguration(TaskManagerServicesConfiguration.java:307)
	at org.apache.flink.runtime.taskexecutor.TaskManagerRunner.startTaskManager(TaskManagerRunner.java:619)
	at org.apache.flink.runtime.minicluster.MiniCluster.startTaskManager(MiniCluster.java:766)
	at org.apache.flink.runtime.minicluster.MiniCluster.startTaskManagers(MiniCluster.java:747)
	at org.apache.flink.runtime.minicluster.MiniCluster.start(MiniCluster.java:459)
	at org.apache.flink.runtime.testutils.MiniClusterResource.startMiniCluster(MiniClusterResource.java:245)
	at org.apache.flink.runtime.testutils.MiniClusterResource.before(MiniClusterResource.java:110)
	at org.apache.flink.runtime.testutils.InternalMiniClusterExtension.beforeAll(InternalMiniClusterExtension.java:72)
	at org.apache.flink.test.junit5.MiniClusterExtension.beforeAll(MiniClusterExtension.java:231)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
Caused by: java.lang.ClassNotFoundException: org.apache.flink.shaded.guava31.com.google.common.collect.Iterables
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 12 more
```